releases/V2.1.0.md

```diff
@@ -0,0 +1,687 @@
+	- [Evolution since previous version](#evolution-since-previous-version)
+		- [Functionnal evolutions](#functionnal-evolutions)
+		- [infrastructure](#infrastructure)
+			- [Development done by User Story (US)](#development-done-by-user-story-us)
+			- [Bugs closed in this version](#bugs-closed-in-this-version)
+		- [production-common](#production-common)
+			- [Development done by User Story (US)](#development-done-by-user-story-us-1)
+			- [Bugs closed in this version](#bugs-closed-in-this-version-1)
+		- [monitoring](#monitoring)
+			- [Development done by User Story (US)](#development-done-by-user-story-us-2)
+			- [Bugs closed in this version](#bugs-closed-in-this-version-2)
+		- [user-web-client](#user-web-client)
+			- [Development done by User Story (US)](#development-done-by-user-story-us-3)
+			- [Bugs closed in this version](#bugs-closed-in-this-version-3)
+		- [processing-sentinel-1](#processing-sentinel-1)
+			- [Development done by User Story (US)](#development-done-by-user-story-us-4)
+			- [Bugs closed in this version](#bugs-closed-in-this-version-4)
+		- [processing-sentinel-2](#processing-sentinel-2)
+			- [Development done by User Story (US)](#development-done-by-user-story-us-5)
+			- [Bugs closed in this version](#bugs-closed-in-this-version-5)
+		- [processing-sentinel-3](#processing-sentinel-3)
+			- [Development done by User Story (US)](#development-done-by-user-story-us-6)
+			- [Bugs closed in this version](#bugs-closed-in-this-version-6)
+	- [Known problems or limitations](#known-problems-or-limitations)
+		- [Bugs closed since the start of the project](#bugs-closed-since-the-start-of-the-project)
+		- [Bugs still opened](#bugs-still-opened)
+	- [Advice for use of the software configuration item](#advice-for-use-of-the-software-configuration-item)
+
+# Introduction
+  
+The purpose of this document is to describe a given software version in terms of known problems, limitations or restrictions with respect to its approved baseline.  
+
+# Software release overview
+  
+| Component | Version tested |
+| :---------- | :----------: |
+| infrastructure | [1.6.0](https://github.com/COPRS/infrastructure/releases/tag/1.6.0) |
+| production-common | [1.15.0](https://github.com/COPRS/production-common/releases/tag/1.15.0) |
+| monitoring | [1.6.0](https://github.com/COPRS/monitoring/releases/tag/1.6.0) |
+| user-web-client | [1.10.0](https://github.com/COPRS/user-web-client/releases/tag/1.10.0) |
+| processing-sentinel-1 | [1.14.0](https://github.com/COPRS/processing-sentinel-1/releases/tag/1.14.0) |
+| processing-sentinel-2 | [1.8.0](https://github.com/COPRS/processing-sentinel-2/releases/tag/1.8.0) |
+| processing-sentinel-3 | [1.14.0](https://github.com/COPRS/processing-sentinel-3/releases/tag/1.14.0) |
+
+# Status of the software configuration item
+
+## Evolution since previous version
+
+### Functionnal evolutions
+  
+Complete production common V1.15.0
```

Comment:
Replace this line by:
Implement CADIP ingestion component
Update distribution components to be compliant with Production Interface Delivery Point_ICD_v1.9 (ESA-EOPG-EOPGC-IF-3)